integrations/woopra/lib/index.js

```diff
@@ -22,14 +22,11 @@ var Woopra = (module.exports = integration('Woopra')
   .option('cookieName', 'wooTracker')
   .option('cookieDomain', null)
   .option('cookiePath', '/')
-  .option('ping', true)
-  .option('pingInterval', 12000)
```

Comment:
I see the ping options are deprecated by the woopra library - just curious, would a user interact with the woopra library any differently if these are turned on?

Reply:
The ping mechanism in Woopra is no longer supported. It has been replaced with a more robust mechanism using Beacons. So the ping and pingInterval configurations are ignored right now by the Woopra Javascript SDK. So, they should be removed from Segment's configuration. 

Users will not need to interact with the library any differently.